numpy/distutils/misc_util.py

```diff
@@ -1687,8 +1687,6 @@ def add_npy_pkg_config(self, template, install_dir, subst_dict=None):
 
         and will be installed as foo.ini in the 'lib' subpath.
 
-        Cross-compilation
-        -----------------
```

Comment:
This is not a valid [docstring section name](https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html#docstring-sections)